CMakeLists.txt

```diff
@@ -187,30 +196,16 @@ MACRO_LOG_FEATURE(JPEG8_FOUND "libjpeg" "JPEG image format support" "http://www.
 IF(JPEG8_FOUND)
     ADD_DEFINITIONS(-DUSE_JPEG)
     ADD_DEFINITIONS(-DUSE_JPEG8)
-    INCLUDE_DIRECTORIES(${JPEG_INCLUDE_DIR})
 ENDIF()
 
 # For registration to libraw_config.h
 MACRO_BOOL_TO_01(JPEG8_FOUND LIBRAW_USE_DNGLOSSYCODEC)
 
-# OpenMP library check
-
-SET(OPENMP_FOUND false)
-
 IF(ENABLE_OPENMP)
-    # OpenMP library detection (requires CMake >= 2.6.3)
-    MACRO_OPTIONAL_FIND_PACKAGE(OpenMP)
+    find_package(OpenMP REQUIRED)
+ENDIF(ENABLE_OPENMP)
 
-    IF(OPENMP_FOUND)
-        SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
-        IF("${OpenMP_CXX_FLAGS}" STREQUAL "-fopenmp")
-            SET(OPENMP_LDFLAGS "-lgomp")
-        ENDIF()
-        IF("${OpenMP_CXX_FLAGS}" STREQUAL "-xopenmp")
-            SET(OPENMP_LDFLAGS "-xopenmp")
-        ENDIF()
-    ENDIF()
-ENDIF()
+set(OPENMP_FOUND TARGET OpenMP::OpenMP_CXX)
```

Comment:
Remove this line?

Reply:
I'm keeping this for now because later it is used 5 times

Reply:
No, the value is not used, only as boolean in "if" statements, and in those you should use `OpenMP_FOUND` which is set by `find_package(OpenMP`.